modules/gdnative/include/gdnative/gdnative.h

```diff
@@ -282,6 +282,8 @@ void GDAPI godot_print_error(const char *p_description, const char *p_function,
 void GDAPI godot_print_warning(const char *p_description, const char *p_function, const char *p_file, int p_line);
 void GDAPI godot_print(const godot_string *p_message);
 
+void GDAPI godot_profiling_add_data(const char *p_signature, uint64_t p_time);
```

Comment:
Same here, should go in the nativescript-specific header instead

Reply:
And this too.